app.rb

```diff
@@ -1,36 +1,87 @@
 require 'webhook_handler'
 require 'octokit'
 require 'open-uri'
+require 'erb'
+
+class FindPopoloFiles
+  def self.from(files)
+    files.find_all { |file| file[:filename].match(/ep-popolo-v1.0\.json$/) }
```

Comment:
We may not want to bind this quite so tightly to the version of the Popolo. This feels like something we could easily overlook when we bump the version of that, and not notice for a while that we're no longer getting notifications…